### ISSUE TITLE
Undefined variable in DES_Heli.sqf

### ISSUE THREAD

**Issue by Bob-Murphy** [official-antistasi-community/A3-Antistasi#223]
```
15:14:28 Error in expression < %1, expected %2, on index %3, in %4", [_par] call _fnc_getTypeName, [_x] call _>
15:14:28   Error position: <_par] call _fnc_getTypeName, [_x] call _>
15:14:28   Error Undefined variable in expression: _par
15:14:28 File A3\functions_f\Debug\fn_errorParamsType.sqf [BIS_fnc_errorParams..., line 146
15:14:28 "armahosts/log: ERROR: [BIS_fnc_relPos] Error: type ANY, expected OBJECT, on index 0, in [any,208.075,323.917]"
15:14:28 Error in expression <is + vehTransportAir);

_posCrashMrk = [_poscrash,random 500,random 360] call BI>
15:14:28   Error position: <_poscrash,random 500,random 360] call BI>
15:14:28   Error Undefined variable in expression: _poscrash
15:14:28 File mpmissions\__cur_mp.chernarus_summer\Missions\DES_Heli.sqf..., line 39
```

Line 14636 in the rpt linked in dev-team channel

**Comment by Toakan** [comment]
These lines need to be swapped. It's initiating the array after it's being used?

https://github.com/official-antistasi-community/A3-Antistasi/blob/master/A3-Antistasi/Missions/DES_Heli.sqf#L39-L40

**Comment by PoweredByPot** [comment]
fixed in #270

**Comment by PoweredByPot** [comment]
done and merged, please close.
